lib/object_3d.dart

```diff
@@ -24,17 +28,21 @@ class Object3D extends StatefulWidget {
   final String? path;
   final String? object;
   final Color color;
+  final double swipeCoef; // pan delta intensity
+  final double dampCoef; // psuedo-friction 0.001-0.999
+  final double maxSpeed; // in rots per 16 ms
 
   @override
   State<Object3D> createState() => _Object3DState();
 }
 
 class _Object3DState extends State<Object3D> {
-  double _angleX = 15.0, _angleY = 45.0;
-  double _previousX = 0.0, _previousY = 0.0;
-
+  double _pitch = 15.0, _yaw = 45.0;
+  double? _previousX, _previousY;
+  double _deltaX = 0.0, _deltaY = 0.0;
   List<Vector3> vertices = [];
   List<List<int>> faces = [];
+  late Timer _updateTimer;
```

Comment:
This Timer has to be cancelled in the `dispose` method

Reply:
noted thanks

Reply:
done